cltk/prosody/old_norse/verse.py

```diff
@@ -589,3 +591,57 @@ def find_alliteration(self):
         :return:
         """
         return Metre.find_alliteration(self)
+
+
+class PoetryTools:
+    def __init__(self, basic=True):
+        if basic:
+            self.syllabifier = Syllabifier(language="old_norse_ipa")
+            self.tr = Transcriber(DIPHTHONGS_IPA, DIPHTHONGS_IPA_class, IPA_class, old_norse_rules)
+            self.tagger = POSTag('old_norse')
+        else:
+            self.syllabifier = None
+            self.phonetic_transcriber = None
+            self.tagger = None
+
+
+class PoeticWord:
```

Comment:
Please add docstring to this and the other class.

Reply:
Done.